source/nodejs/adaptivecards/package.json

```diff
@@ -56,5 +57,8 @@
 			"jsx",
 			"json"
 		]
+	},
+	"dependencies": {
+		"@ms/1ds-analytics-web-js": "^2.0.4"
```

Comment:
Maybe have this be an "optionalDependency" ([https://docs.npmjs.com/files/package.json#optionaldependencies](https://docs.npmjs.com/files/package.json#optionaldependencies)) to avoid breaking installs for the golden path where this provider is not going to be initialized (which at this point is never by default)

Reply:
Good idea, I think this may fix the issue we are having